src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -736,13 +741,13 @@ std::string CascadiaSettings::_ApplyFirstRunChangesToSettingsTemplate(std::strin
 // - a non-owning pointer to the scheme.
 const ColorScheme CascadiaSettings::GetColorSchemeForProfile(const GUID profileGuid) const
 {
-    auto* profile = FindProfile(profileGuid);
-    if (!profile)
+    auto profile = FindProfile(profileGuid);
+    if (profile == nullptr)
     {
         return nullptr;
     }
-    auto schemeName = profile->GetSchemeName().has_value() ? profile->GetSchemeName().value() : L"\0";
-    auto scheme = _globals.GetColorSchemes().find(schemeName);
+    auto schemeName = !profile.ColorSchemeName().empty() ? profile.ColorSchemeName() : L"\0";
+    auto scheme = _globals.GetColorSchemes().find(schemeName.c_str());

```

Comment:
please don't use `c_str()` unless you have _no other option_

Reply:
if you're passing this in place of a wstring_view, use `std::wstring_view{ schemeName }`